common/lib/xmodule/xmodule/mongo_utils.py

```diff
@@ -58,9 +58,14 @@ def connect_to_mongodb(
             wait_time=retry_wait_time
         )
 
+    # default the authSource to be whatever db we are connecting to (for backwards compatiblity)
+    authSource = db
```

Comment:
This is more succinct and easier to read than the `if` statement.
```suggestion
    authSource = kwargs.get('authSource', db)
```